### PR TITLE
Set up nginx buildpack and deployment of app as static resource.

### PR DESCRIPTION
**Pull request by rpiaggio**

### CODE REVIEW

**Comment by cquiroz** [comment]
We probably shouldn't commit `.DS_Store` files

File: static.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (cquiroz) [inline static.json]: Is this nginx configuration?

Comment (rpiaggio) [reply static.json]: The static server Heroku buildpack turns this into nginx configuration. So, indirectly, yes.

**Comment by rpiaggio** [comment]
> We probably shouldn't commit `.DS_Store` files

Fixed.

File: HEROKU.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
-In order to deploy to Heroku, the Heroku app must be configured to provision `node` besides the `Scala` environment.
+In order to deploy a Scala.js app to Heroku, the Heroku app must be configured to provision `node` besides the `Scala` environment.
```
Comment (cquiroz) [inline HEROKU.md]: Great to have documentation

File: package.json [hunk 3]
```diff
@@ -1 +1,8 @@
-{ "engines": { "node": "12.14.1" } }
\ No newline at end of file
+{ 
+    "engines": { 
+        "node": "12.14.1" 
+    },
+    "scripts": {
+        "test": "./dummy-script"
```
Comment (cquiroz) [inline package.json]: Is this script in the repo?

Comment (rpiaggio) [reply package.json]: Yes, had to add it.

**Pull request merged**